index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Clone</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <!--Grid starts-->
+    <div class="grid">
+      <!--Left Side-Bar Starts-->
+      <div class="left-bar">
+        <!--Twitter Logo on left-bar-->
+        <div class="logo-grid-leftbar">
+          
+          <img src="https://upload.wikimedia.org/wikipedia/commons/4/4f/Twitter-logo.svg">
+        </div>
+
+        <!--Options on left-bar-->
+        <div class="options-nav">
+          <div class="home"><h2>Home</h2></div>
+          <div class="explore"><h2>Explore</h2></div>
+          <div class="notifications"><h2>Notifications</h2></div>
+          <div class="messages"><h2>Messages</h2></div>
+          <div class="bookmarks"><h2>Bookmarks</h2></div>
+          <div class="lists"><h2>Lists</h2></div>
+          <div class="profile"><h2>Profile</h2></div>
+          <div class="more"><h2>More</h2></div>
+        </div>
+        <!--Tweet button on left-bar-->
+        <div class="tweet-grid">
+          <button class="tweet-button1"><strong>Tweet</strong></button>
+        </div>
+        <!--Closing Left Side-Bar - Div below-->
+      </div>
+      <!--Left Side-Bar Ends-->
+
+      <!--Main Bar Starts-->
+      <div class="main">
+        <div class="home-page"><h2>Home</h2></div>
+        <div class="posting-section">
+            <div class="whats-happening"><p>What's Happening?</p></div>
+            <div class="uploading">
+            <div class="media-upload">
+              <img src="./image-outline-icon.svg">
+            </div>
+            <div class="gif-upload">
+              <img src="./gif-file-icon.svg">
+            </div>
+            <div class="poll-upload">
+              <img src="./poll-icon.svg">
+            </div>
+            <div class="emoji-upload">
+              <img src="./cool-emoji-icon.svg">
+            </div>
+            <div class="schedule-upload">
+              <img src="./daily-schedule-icon.svg">
+            </div>
+          </div>
+          <!--Add div for button-->
+            <button class="tweet-button2">Tweet</button>
+       </div>
+       <hr>
+       <!--Main - last section feed1-->
+       <div class="feed">
+        <div class="feed-grid1-profile-pic">
+          <img src="./default-profile-picture-grey-female-icon.svg">
```

Comment:
Inserted an icon SVG instead of an img, when I did insert an img - I was unable to downsize, I believe the type of img was a png, is this a correct file type?